pkg/minishift/config/config.go

```diff
@@ -27,12 +27,14 @@ var Config *InstanceConfig
 type InstanceConfig struct {
 	FilePath string `json:"-"`
 	OcPath   string
+
+	HostFolders []HostFolder
 }
 
 // Create new object with data if file exists or
 // Create json file and return object if doesn't exists
 func NewInstanceConfig(path string) (*InstanceConfig, error) {
-	cfg := new(InstanceConfig)
+	cfg := &InstanceConfig{HostFolders: []HostFolder{}}
```

Comment:
I am wondering whether it is the right thing to add this here? There is also the config.json in ~/.minishift/config which contains the other configuration values like cpus, memory etc. I am wondering whether this config should not go there (we might need to add some helper methods to read/write to this config similar to what we have here).

The reason I am asking is, that this config is specific to the instance. It also gets deleted with the instance. So if you do a `minishift delete` you need to re-create the settings. Settings in ~/.minishift/config/config.json on the other hand persist across VM deletion which I think is more what a user would like to do. WDYT?